packages/core/Earthfile

```diff
@@ -0,0 +1,25 @@
+VERSION 0.6
+
+prepare:
+  FROM node:14-alpine
+  WORKDIR /app
+  COPY package.json package-lock.json .
+  RUN npm install --shrinkwrap
+  COPY --dir \
+       rollup.config.js \
+       tsconfig.json \
+       src \
+       index.ts \
+       types.d.ts \
+       .
+
+build:
+  FROM +prepare
+  RUN npm run build
+  SAVE ARTIFACT dist AS LOCAL dist
+  SAVE IMAGE builder.io/core:latest
+
+test:
+  FROM +prepare
+  COPY jest.config.ts .
+  RUN npm run test
```

Comment:
👀 Mind outlining what this is exactly? Is it a replacement for npm scripts?

Reply:
It's like docker - it doesn't necessarily have be in this PR but it will be a part of the internal build system and if I have to build these packages it will be required at some point.

tl;dr lets you build a docker image and documents what the steps are to build this package.